runtime/stdlib/contracts/test.cdc

```diff
@@ -2,160 +2,154 @@
 ///
 pub contract Test {
 
-    /// Blockchain emulates a real network.
+    /// backend emulates a real network.
     ///
-    pub struct Blockchain {
+    pub let backend: AnyStruct{BlockchainBackend}
```

Comment:
Can this be `access(self)`?
```suggestion
    access(self) let backend: AnyStruct{BlockchainBackend}
```

Reply:
Updated in https://github.com/onflow/cadence/pull/2783/commits/72bbfe1e5d4a5ed438762cfe05a6c7b1c1f731ef